build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.projectlombok:lombok:1.18.18'
 	implementation 'org.json:json:20211205'
+	implementation 'com.squareup.okhttp3:okhttp:3.14.6'
```

Comment:
Are you using okhttp to make the HTTP requests? It's unclear to me that this is being used.